e2e-tests/ukef/cypress/support/external-api/api.js

```diff
@@ -0,0 +1,7 @@
+// mock out number generator call and use 7 digit ID instead
+// reduced in size from 10 as premium schedule cannot handle large numbers
+let ukefId = 1e7;
```

Comment:
@ttbarnes I imagine the commonisation of test elements might be the next big chunk of tech debt we wish to face as it's probably the one that I'm seeing gain the most interest on (with the exception of the need for builders for deals/facilities)?

Reply:
DRY test mocks would be great

Also keen to replace all CommonJS imports with ES6 imports/exports